dom-weather-app/public/app.js

```diff
@@ -0,0 +1,92 @@
+const cityInput = document.querySelector('#city-input');
+const startButton = document.querySelector('#startButton');
+
+const apiKey = 'kUfkPA9VVU5407oDJWDoh2T10ZALToRm';
+
+const allWeatherInformation = async () => {
+    const cityDetails = await getCity(cityInput.value);
+    const weatherConditions = await getWeather(cityDetails.cityID);
+
+    return {
+        isDay: weatherConditions[0].IsDaylight,
+        temperatureInFahrenheits: weatherConditions[0].Temperature.Value,
+        iconPhrase: weatherConditions[0].IconPhrase,
+        weatherIcon: weatherConditions[0].WeatherIcon
+    }
+}
+
+startButton.addEventListener('click', async () => {
+    updateUI(
+        (await allWeatherInformation()).isDay,
```

Comment:
to robi 4x requesty :D zrób jeden await na górze i przypisz do zmiennej i całą zmienną przekaż
```
const data = await allWeatherInformation();
updateUI(data),
 ```

Reply:
Fuck -.- poprawione, dzięki